### PR TITLE
added buttons to click on to map page

### PR DESCRIPTION
**Pull request by induramesh**
Each map marker now has a button. Clicking the button opens up a new page  in a new tab (placeholder), parks/objectID

### CODE REVIEW

File: config/urls.py [hunk 1]
```diff
@@ -24,4 +24,5 @@
     path('', TemplateView.as_view(template_name='home.html'), name='home'), # new
     path('accounts/signup/', TemplateView.as_view(template_name='signup.html'), name='signup'), # new
     path('maps/', include('maps.urls')),
+    path('parks/', include('parks.urls')),
```
Comment (susmitabhowmik) [inline config/urls.py:27]: Not all the facilities are parks, I am wondering if this URL should be `facility/` instead? Let me know what you think!

File: parks/views.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+from django.http import HttpResponse
+
+def show(request, id):
+    return HttpResponse(f'this will be the page for the park with mapbox objectid: { id }')
```
Comment (susmitabhowmik) [inline parks/views.py:4]: Awesome work connecting the location id with this page!

File: config/urls.py (hunk 1 above)
Comment (induramesh) [reply config/urls.py:27]: Since it's such a minor thing, can we deploy this for now and leave it as a nice to have?

Comment (induramesh) [reply config/urls.py]: Done :)

**Pull request merged**